modules/system/models/MailSetting.php

```diff
@@ -72,9 +73,16 @@ public function initSettingsData()
         $this->smtp_authorization = !!strlen($this->smtp_user);
     }
 
+    public function getFailoverMailersOptions()
+    {
+        $config = App::make('config');
+        return array_keys($config->get('mail.mailers'));
```

Comment:
```suggestion
        $mailers = App::make('config')->get('mail.mailers');
        unset($mailers['failover']);
        return array_keys($mailers);
```
To avoid having `failover` in the list of transports.